client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/email-validation-banner.tsx

```diff
@@ -0,0 +1,55 @@
+import { Gridicon } from '@automattic/components';
```

Comment:
Could we add unit tests for this in a follow-up?

Reply:
Definitely! And I could use some help since when I tried rendering the step-content component with jest I was getting so many errors, one after another...